sys/Makefile.dep

```diff
@@ -853,4 +853,8 @@ ifneq (,$(filter fido2_ctap,$(USEMODULE)))
   USEMODULE += fido2
 endif
 
+ifneq (,$(filter rtt_rtc,$(USEMODULE)))
+  USEMODULE += rtc_utils
```

Comment:
Couldn't this be added inside `drivers/rtt_rtc/Makefile.dep` instead ?

Reply:
Hmm yeah not sure why added it there